content/blog/elao/importance-design-system-pour-UX-efficaces.md

```diff
@@ -0,0 +1,205 @@
+
+Aujourd’hui, designers et développeurs doivent prendre en compte de nombreux paramètres 
+afin de s’assurer que leurs produits digitaux respectent les attentes et besoins de leurs utilisateurs 
+tout en s’articulant autour de diverses contraintes. 
+
+Ces dernières peuvent rendre la création d’applications, de sites et autres produits digitaux
+plus complexe qu’auparavant, impliquant souvent des évolutions fréquentes 
+notamment dans le cadre d’un processus itératif - ainsi que des allers et retours entre équipes
+(designers/développeurs). 
+
+On sait que cela peut engendrer des risques pour le projet (perte d’information, confusions, …). 
+
+Il n’existe pas de solution magique pour pallier complètement ces potentiels problèmes, mais dans ce contexte, 
+le design system est une solution multifacette pouvant aider les équipes à s’organiser 
+afin de mener au mieux le développement de leurs produits.
+
+## 1 - Mais c'est quoi un design system ?
+
+À ne pas confondre avec les design patterns, les style guides ou encore les bibliothèques de composants, 
+un design system est un référentiel UX et UI, principalement utilisé par les designers et développeurs 
+(mais il peut être utile à tous les membres d’équipe) 
+dans le cadre de la conception et du développement de d’applications web et mobile.
+
+La création d’un design system est motivée par des objectifs de cohérence des interfaces utilisateur, 
+de maintenabilité du design ainsi que du  code et de collaboration entre les équipes de designers et développeurs.
+
+En tant que référentiel, ce dispositif se fonde sur l’Atomic Design : 
+on crée les éléments de conception de base (comme les couleurs, les polices, etc.) 
+et on les assemble au fur et à mesure pour créer des composants de plus en plus complexes. 
+Un peu comme des legos ! Le design system harmonise les règles de design et de comportement des composants, 
+instaurant ainsi un langage commun entre designers et développeurs. 
+
+Autrement dit, le design system est une sorte de boîte à outils UX/UI, un set d’éléments variés, standardisés, 
+assemblables et réutilisables. 
+Je disais en début d’article qu’il fallait veiller à ne pas confondre design patterns, style guides, 
+bibliothèques de composants et design system ; c’est parce que le design system inclut, entre autres, 
+tous ces éléments et bien d’autres que nous allons voir très vite !
+
+## 2 - Construire un design system efficace
+
+### 2.1 - Quels éléments y inclue-t-on ?
+
+Chaque design system est propre à l’entreprise qui le crée et à ses besoins. 
+Des designs systems pour des projets d’envergure moindre pourront par exemple inclure seulement un kit UI détaillé et 
+quelques éléments graphiques et/ou de développement, là où certains vont bien plus loin en y incluant par exemple 
+des bibliothèques de patterns et une documentation poussée.
+
+Une étude menée par Google et Clarity en 2020 auprès d’utilisateurs et concepteurs de design systems a montré que 
```

Comment:
@ameliedefrance peux-tu ajouter le lien de l'étude ? 
https://material.io/blog/research-state-of-design-systems-2020